april_tag_updates/docking_script.py

```diff
@@ -0,0 +1,127 @@
+expected_dist = 37
+known_width = 4.18 ##inches
+focal_length = 1.37478492e+03
+from electrical.motor_controller import BasicMotorController
+import numpy as np
+import cv2
+import time 
+from enum import Enum
+
+arucoDict = cv2.aruco.Dictionary_get(cv2.aruco.DICT_APRILTAG_36h11)
+arucoParams = cv2.aruco.DetectorParameters_create()
+cap = cv2.VideoCapture(0)
+window = 'Camera'
+cv2.namedWindow(window)
+def distance_to_camera(knownWidth, focalLength, perWidth):
+  # compute and return the distance from the maker to the camera
+  return (knownWidth * focalLength) / perWidth
+
+def direction_of_tag(corner,image):
+  (_, _, _, _) = corner
+  _,w = image.shape[0], image.shape[1]
+  cx, _ = int(960),int(540)
+  if  bottomLeft[0] > cx:
+    return "right"
+  else:
+    return "left"
+
+idDict = {1: "R1", 2: "R2", 3: "R3"}#TODO add all the other tags
+visited = []
+timeout =  10
+angle_zero = 10
```

Comment:
units? what's angle_zero?

Reply:
angle_zero is in degrees. Our camera is not accurate enough to actually be perfectly aligned with the center of the tag aka when the angle would be 0 degrees. angle_zero is 10 since we'll say if angle is 10, then the center of the camera is perfectly aligned with the center of the tag.

Reply:
Also I renamed this to angle_min to be more intuitive.

Reply:
sounds good! a comment might be nice as well